apps/beets-frontend-v3/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "pnpm --filter @repo/lib graphql:gen && next build",
     "dev": "concurrently \"pnpm --filter @repo/lib graphql:gen --watch\" \"next dev -p 3001\"",
+    "dev:mock": "next dev -p 3001",
```

Comment:
I guess that you use this locally with your mock setup but the name of the script could be confusing for someone external. Maybe better calling it something like `dev:without-graphql-gen` or `dev:no-gen`?

Reply:
Renamed them